src/components/Dashboard.tsx

```diff
@@ -94,38 +85,89 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
   )
 }
 
-const FilterComponent: React.FC<{
+const PanelComponent: React.FC<{
   template: TemplateDetails
-  filter: Filter
-  setFiltersCount: Function
-}> = ({ template, filter, setFiltersCount: setLoadingFilters }) => {
+  filters: Filter[]
+}> = ({ template, filters }) => {
   const templateType = template.code
-  const { loading, applicationCount } = useListApplications({
-    type: templateType,
-    perPage: 1,
-    ...filter.query,
-  })
+  const [loadedFiltersCount, setLoadedFiltersCount] = useState(0)
```

Comment:
Hmmm, I think we *should* be able to get rid of "loadedFiltersCount". That's something I asked Fergus to add when he added the Loading spinner to these components, as it was the easiest way to keep track of when all the filter queries are finished.

But now we're just doing one query per panel, so I don't see why we'd need to keep a count anymore -- the `loading` value from `useListApplications` should be all you need.